android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Interactor.kt

```diff
@@ -56,11 +59,18 @@ abstract class Interactor<C : Parcelable, V : RibView>(
     val isAttached: Boolean
         get() = behaviorRelay.value == ACTIVE
 
+    internal var tag = "${this::class.java.name}.${UUID.randomUUID()}"
```

Comment:
Probably, we should reuse the tag from the node to be consistent

Reply:
I tried that first, but there's a problem: if you need to setup a subscription in your `Interactor`, then it's `id()` method will be invoked, and since `Node` is lateinit (will be created after `Interactor`), it crashes if it tries to forward the call. So `Interactor` needs to identify itself regardless of `Node`.

My idea is that we can remove `tag` from `Node` altogether.

Reply:
Alright, probably makes sense

Reply:
Also an idea for future PR to unify identifiers (there's one field called `identifier` in `Node` that extends `Rib` interface, there's `tag`, theres `ribID`). So probably `identifier: Rib` could be extracted and injected both to `Node` and to `Interactor`, and the UUID mechanism could be implemented there. Maybe.

Reply:
What the solution?
Are you gonna remove `tag` from `Node` together?